crates/core_arch/src/aarch64/neon/generated.rs

```diff
@@ -15,7 +15,7 @@ use stdarch_test::assert_instr;
 #[inline]
 #[target_feature(enable = "neon,sha3")]
 #[cfg_attr(test, assert_instr(eor3))]
-#[unstable(feature = "stdarch_neon_sha3", issue = "117225")]
+#[stable(feature = "stdarch_neon_sha3", since = "CURRENT_RUSTC_VERSION")]
```

Comment:
Is `CURRENT_RUSTC_VERSION` supported in submodules?

Reply:
I wasn't sure exactly what to put for this. `CURRENT_RUSTC_VERSION` was cargo culted from #1399, which performed a similar stabilization.

Reply:
No, it's not supported. You should use the current nightly version.

Reply:
Changed to `1.78.0` in 4bc5e32